### PR TITLE
Validate billable email when updating account

### PR DESCRIPTION
**Pull request by malandrina**
* Update `AccountsController#update` to validate billable email
* Add `email_validator` to Gemfile
* Rename `AccountsController` spec file
* Don't use `strong_parameters` in `AccountsController` because it doesn't not return a JSON-encoded response with nicely formatted error messages, per this issue: https://github.com/rails/strong_parameters/issues/157

### CODE REVIEW

**Comment by thorncp** [comment]
LGTM :+1:

File: app/controllers/accounts_controller.rb [hunk 1]
```diff
@@ -22,8 +24,15 @@ def update
 
   private
 
+  def validate_billable_email
+    unless EmailValidator.valid?(new_billable_email)
+      error_message = I18n.t("account.email.invalid")
```
Comment (Koronen) [inline app/controllers/accounts_controller.rb]: `error_message = I18n.t("account.billable_email.invalid")`?

Comment (teoljungberg) [inline app/controllers/accounts_controller.rb]: I prefer `billable_email` over just plain 'old `email` aswell

Comment (malandrina) [inline app/controllers/accounts_controller.rb]: Good call. Changed in my most recent commit.

**Pull request merged**